internal/provider/test_constants.go

```diff
@@ -88,3 +87,50 @@ resource "aci_l3out_loopback_interface_profile" "test" {
   addr           = "1.2.3.5"
 }
 `
+
+const testConfigDefaultMin = ``
```

Comment:
what is this for?

Reply:
This was a work around to avoid adding additional conditionals in the template. Removed it!